requirements/pinned.txt

```diff
@@ -1,10 +1,24 @@
-certifi==2024.12.14        # via requests
-cffi==1.17.1              # via cryptography, pynacl
-charset-normalizer==3.4.0 # via requests
-cryptography==44.0.0      # via securesystemslib
-idna==3.10                 # via requests
-pycparser==2.22           # via cffi
-pynacl==1.5.0             # via securesystemslib
+#
+# This file is autogenerated by pip-compile with Python 3.11
+# by the following command:
+#
+#    pip-compile --output-file=requirements/pinned.txt --strip-extras requirements/main.txt
+#
+certifi==2024.12.14
+    # via requests
+cffi==1.17.1
+    # via cryptography
+charset-normalizer==3.4.0
+    # via requests
+cryptography==44.0.0
+    # via securesystemslib
+idna==3.10
+    # via requests
+pycparser==2.22
+    # via cffi
 requests==2.32.3
-securesystemslib[crypto,pynacl]==1.2.0
-urllib3==2.2.3           # via requests
+    # via -r requirements/main.txt
+securesystemslib==1.2.0
+    # via -r requirements/main.txt
```

Comment:
the extras are no longer documented in this file since pip-compile is changing the default output to be compatible with "--constraints": this seems sensible.

I'm explicitly using `pip-compile --strip-extras` as the command so we won't get bitten by one version of pip-compile having a different default than another version at some point in future